libweasyl/libweasyl/models/tables.py

```diff
@@ -664,6 +664,21 @@ def default_fkey(*args, **kwargs):
 Index('ind_searchmapsubmit_targetid', searchmapsubmit.c.targetid)
 
 
+searchmapartist = Table(
+    'searchmapartist', metadata,
+    Column('tagid', Integer(), primary_key=True, nullable=False),
+    Column('targetid', Integer(), primary_key=True, nullable=False),
+    Column('settings', CharSettingsColumn({
```

Comment:
What would be the particular benefits of splitting them out? One of the reasons I went with this approach was that I considered it wouldn't be logically possible for someone to have the same tag in both lists, but beyond that I was mostly just mirroring the other searchmap* tables